applications/xcm-tools-follow-up.md

```diff
@@ -0,0 +1,111 @@
+# XCM Tools follow-up
+
+* **Team Name:** [gmajor](https://github.com/gmajor-encrypt)
+* **Payment Address:** 0xC3094f0ddce699a1Ad9Ef2621DF68Cd297a4c44F(USDC)
```

Comment:
Please change this to an AssetHub address. We no longer support USDC on Ethereum.